aws-athena/views/reporting-vw_ratio_stats.sql

```diff
@@ -38,18 +38,45 @@ town_names AS (
     FROM {{ source('spatial', 'township') }}
 ),
 
+final_model_parsed AS (
```

Comment:
As an aside, I kind of wish there was some way to parse or process seeds so we wouldn't need to use this CTE. We _could_ just make another model downstream of the seed, but that feels a little messy. Thoughts?

Reply:
We can do some naive parsing at the level of the raw seed by specifying [`column_types`](https://docs.getdbt.com/reference/resource-configs/column_types), but that won't help with encoding the `triad_name` categoricals, and I'm not super confident it would work with the `township_code_coverage` array field either. I think a transformation model is actually the idiomatic way of doing this in dbt, since dbt encourages explicit data transformations; an ephemeral model on top of the raw seed would be ideal if ephemeral models were supported in dbt-athena (https://github.com/ccao-data/data-architecture/issues/238), but I think if we renamed the seed model to something like `model.final_model_raw` and made `model.final_model` a view on top of that then it would still be pretty clear what's going on.

If we think that this particular parsing of the seed will only be used in a couple places, I'm also fine keeping the code as-is and using a CTE for parsing where necessary. A transformation model is only necessary to the extent that this logic will be widely used, in my opinion.

Reply:
I think it's valuable to have a parsed version available. Added it in 7011d36917c407dbf946a0aa24f85f8fb7be415b.